drivers/soc/atmel/soc.c

```diff
@@ -237,7 +243,8 @@ struct soc_device * __init at91_soc_init(const struct at91_soc *socs)
 	}
 
 	if (!soc->name) {
-		pr_warn("Could not find matching SoC description\n");
+		pr_warn("Could not find matching SoC description"
+			" (cidr %08x, exid %08x)", cidr, exid);
 		return NULL;
```

Comment:
This is an unrelated change

Reply:
Should I revert the change of the warning and probably create a different pull request? It is definitely useful, when running on yet unsupported soc's.

Reply:
Yes, two separate patches would be the best. However, This will increase the kernel size for a very unusual use case. Most of the time, the soc ids will be present. Maybe use pr_debug ?

BTW, are you actually using this driver for something useful?

Reply:
I reverted the change of the print and decided not to create a different pull request for it. As You pointed out, it should be of rare use. To be honest, I don't know, what the soc registration is good for. I just stumbled over the ""Could not find matching SoC description" warning in the dmesg and fixed it.

You proposed to submit this change upstream. I guess You mean submitting it to the mainline kernel. I requested the pull to linux-at91, because this is my direct upstream. And I thought, that the linux-at91 maintainers will submit relevant changes further up.

Reply:
Yes, we can do it.